storage/src/database/mod.rs

```diff
@@ -50,6 +51,11 @@ impl<B: Backend, Sch: Schema> Storage<B, Sch> {
         })
     }
 
+    /// Dump raw database contents into a data structure
+    pub fn dump_all(&self) -> crate::Result<raw::RawDb> {
```

Comment:
Could you also add this method to `chainstate-storage`? So that it can be used in test-suite

Reply:
I renamed the method to `dump_raw` and exposed it in `chainstate_storage::Store`. Please have a look.